crates/rooch-indexer/migrations/2023-10-31-085226_transactions/up.sql

```diff
@@ -17,7 +18,7 @@ CREATE TABLE transactions (
     state_root                          VARCHAR      NOT NULL,
     event_root                          VARCHAR      NOT NULL,
     gas_used                            BIGINT       NOT NULL,
-    status                              VARCHAR      NOT NULL,
+    tx_execute_status                   VARCHAR      NOT NULL,
```

Comment:
Why change the name?

Reply:
The meaning of `status` in the database is not clear enough. The disadvantage is that the fields in SQL are inconsistent with the original Struct fields.

Reply:
If the status is not a keyword for SQL, I prefer to keep it consistent with Rust Struct. Because it is in the transaction table, the status is transaction status and is clear.

Reply:
Yeah, the merge version has already changed back to `status`